lua/super-commit/window.lua

```diff
@@ -10,8 +10,17 @@ function M.set_autocmd()
     pattern = '*.super',
     -- pattern = 'COMMIT_EDITMSG',
     callback = function()
-      local bufnum_table = window_open.double_vertical()
-      git_cmds.show_git_status(bufnum_table[3])
+      local bufnum_table = window_open.setup()
+      local commands = git_cmds.commands
```

Comment:
```suggestion
      local command_table = git_cmds.commands
```
Name variables consistently.

Reply:
Fixed in d6019e61631eea4948376d76d80b44c8aff49116.

Reply:
This is not mandatory and we can fix it in a different PR.